tests/drivers/watchdog/wdt_basic_api/boards/stm32_wwdg_h7.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&wwdg1 {
```

Comment:
In order to simplify things, I'd propose to update stm32h7.dtsi definition with:
```
		wwdg: wwdg1: watchdog@50003000 {
```
This will enable use of wwdg on this series as well.
Same for iwdg.